dags/openshift_nightlies/config/install/hypershift/none-type.json

```diff
@@ -1,9 +1,10 @@
 {
     "cluster_install_method": "osd",
-    "ocm_cli_version": "master",
```

Comment:
I use to point to "container" on rosa when I dont want to use master, so we dont have on the default behavior. 

https://github.com/cloud-bulldozer/airflow-kubernetes/blob/master/dags/openshift_nightlies/scripts/install/rosa.sh#L201

It is better than comparing to null, which could be confuse

Reply:
Sure I will set a default value instead of `null`.